### PR TITLE
Shortpass demo feedback

### PR DESCRIPTION
**Pull request by LizaFeng**
Not deployed yet. Please test locally. When a new user is registering for an account and their password is too short, there will be a message to tell them that they need a longer password.

### CODE REVIEW

**Review by Lindanny9721 — APPROVED** [review]
Looks like it works.

**Review by Annie-Mai — APPROVED** [review]
Looks like its working

**Comment by LizaFeng** [comment]
After getting approval, im merging this branch

**Pull request merged**
